pkg/cluster/cluster.go

```diff
@@ -268,11 +271,14 @@ func (c *Cluster) run(stopC <-chan struct{}) {
 				c.status.Control()
 			}
 
-			running, pending, err := c.pollPods()
+			running, pending, readyPods, unreadyPods, err := c.pollPods()
```

Comment:
Why not just return ready and unready?

We use replace the logic of `pending` with `unready`. `running` with `ready`.